src/pages/MenuPage.ts

```diff
@@ -5,24 +5,44 @@ export class MenuPage extends AbstractPage {
   readonly page: Page;
   readonly url: string;
   readonly coffeeItemLocators: Locator;
+  readonly addToCartConfirmButton: Locator;
+  readonly addToCartRejectButton: Locator;
 
   constructor(page: Page) {
     super(page);
     this.page = page;
     this.url = ''; 
     this.coffeeItemLocators = page.locator(`li:has(h4)`);
+    this.addToCartConfirmButton = page.getByRole('button', { name: 'Yes'});
+    this.addToCartRejectButton = page.getByRole('button', { name: 'No'});
   }
 
   async getPrice(name: string): Promise<number> {
     test.step(`Gets the price of ${name} cup`, async() => {});
     const headerString = await this.coffeeItemLocators.locator(`h4:has-text("${name} $")`).textContent() || '';
     return +headerString.split(' $')[1];
   }
-
+  
   async addToCart(name: string): Promise<void> {
-    await test.step(`Adds ${name} item to cart`, async() => {
+    await test.step(`Adds ${name} item to cart by left click`, async() => {
```

Comment:
It's better to name step just like you would do in the manual test case so that its easier to read in the report:
`Add ${name} item to cart by left click';